Jenkinsfile

```diff
@@ -17,7 +18,7 @@ properties([
 /**
  * Main pipeline definition for building the engine.
  *
- * It uses the Scripted Pipeline Syntax.
+ * It uses the Declarative Pipeline Syntax.
  * See https://www.jenkins.io/doc/book/pipeline/#declarative-versus-scripted-pipeline-syntax
```

Comment:
Since it uses declarative syntax now, I think we can now better link to https://www.jenkins.io/doc/book/pipeline/syntax/